fastchat/serve/gradio_block_arena_vision_anony.py

```diff
@@ -471,10 +566,10 @@ def build_side_by_side_vision_ui_anony(context: Context, random_questions=None):
         )
 
         multimodal_textbox = gr.MultimodalTextbox(
-            file_types=["image"],
+            file_types=["file"],
```

Comment:
why do we need to change this?

Reply:
When I add ["image", "application/pdf"], it doesn't let me load pdf and if I add ["image", "pdf"], it gives me an error that it's not "application/pdf". I am still trying to find a fix but temporarily just allowed for all file types for testing.

Reply:
I am checking if the file type is pdf or image in the add_text function and raising error if not.